vidur/simulator.py

```diff
@@ -35,7 +35,9 @@ def __init__(self, config: SimulationConfig) -> None:
             self._config.request_generator_config,
         )
         self._metric_store = MetricsStore(
-            self._config.metrics_config, self._config.cluster_config
+            config=self._config.metrics_config,
+            cluster_config=self._config.cluster_config,
+            config_dict=self._config.to_dict(),
```

Comment:
may be let's just pass the config to metricsstore directly?

Reply:
Updated to directly use SimulationConfig